source/common/protobuf/utility.cc

```diff
@@ -106,6 +107,60 @@ absl::Status validateDurationAsMillisecondsNoThrow(const ProtobufWkt::Duration&
   return validateDurationNoThrow(duration, kMaxInt64Nanoseconds);
 }
 
+using JsonStringStreamer = Json::StreamerBase<Json::StringOutput>;
```

Comment:
Please move this to the end of json_streamer.h, next to the nickname for the buffered streamer. You can name it StringStreamer since it will be in the 'Json' namespace.

We should do a follow-up PR to rename Json::Streamer to Json::BufferStreamer for consistency.

Reply:
SGTM.